netsim/extra/ebgp.multihop/junos.j2

```diff
@@ -0,0 +1,43 @@
+{# global #}
+protocols {
+  bgp {
+    group ebgp-peers {
+{% for n in bgp.neighbors if n.type == 'ebgp' and n.multihop is defined %}
+{%   for af in ['ipv4','ipv6'] if n[af] is defined %}
+      neighbor {{ n[af] }} {
+        multihop;
+{%     if n._source_lb_data[af] is defined %}
+        local-address {{ n._source_lb_data[af]|ipaddr('address') }}
```

Comment:
Would suggest to only add ```n._local_address```, not the entire loopback interface

Reply:
Potentially, we need IPv4 and IPv6 addresses, so we could either copy those from loopback data into a new variable, or just add a pointer to the loopback data (which is what you did). Your choice is simpler ;) and one of these days I might clean things up and change all the remaining templates to use just the loopback interface data.